result.go

```diff
@@ -381,17 +381,42 @@ func (q Query) ToMapBy(
 	res.Elem().Set(m)
 }
 
-// ToSlice iterates over a collection and populates the result slice with elements.
+// ToSlice iterates over a collection, copy to result slice the collection elements,
```

Comment:
@cleitonmarx I now addreesed this. Please take a look at the new `ToSlice()` doc.
cc: @kalaninja  as well.